Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            Console.WriteLine("What is your name and surname?");
+            string name1 = Console.ReadLine();
+            Console.WriteLine("How old are you?");
+            string age1 = Console.ReadLine();
+            Console.WriteLine("How much do you weigh?");
+            double weight1 = double.Parse(Console.ReadLine());
+            Console.WriteLine("How tall are you? Answer in CM.");
+            double height1 = double.Parse(Console.ReadLine());
+
+            double bmi1 = weight1 / ( (height1 / 100.0) * (height1 / 100.0) );
+            Console.WriteLine("");
+
+            Console.WriteLine($"{name1} is {age1} years old, his weight is {weight1} kg, and his height is {height1}.");
+            Console.WriteLine($"His BMI is {bmi1}.");
+
+            Console.Write("Now it is time for another one to write their info!");
+            Console.WriteLine("");
+
+
+            Console.WriteLine("What is your name and surname?");
+            string name2 = Console.ReadLine();
+            Console.WriteLine("How old are you?");
+            string age2 = Console.ReadLine();
+            Console.WriteLine("How much do you weigh?");
+            double weight2 = double.Parse(Console.ReadLine());
+            Console.WriteLine("How tall are you? Answer in CM.");
+            double height2 = double.Parse(Console.ReadLine());
+
+            double bmi2 = weight2 / ( (height2 / 100.0 ) * (height2 / 100.0) );

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'bmi2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836739)